lib/pure/random.nim

```diff
@@ -563,17 +562,19 @@ proc initRand*(seed: int64): Rand =
 
     let now = getTime()
     var r2 = initRand(now.toUnix * 1_000_000_000 + now.nanosecond)
-
-  doAssert seed != 0 # 0 causes `rand(int)` to always return 0 for example.
+  let seed = if seed != 0: seed else: 1 # because 0 is a fixed point
```

Comment:
This should be documented (also doesn't seem related to the title of this PR).

Reply:
done (and fixed title)